### PR TITLE
use broccoli-caching-writer in stead of broccoli-writer

### PR DESCRIPTION
**Pull request by pieter-v**
Using the caching writer reduces the rebuild times if runing in serve mode

### CODE REVIEW

**Comment by dfournier** [comment]
Hi!

Thanks for the patch, it's a good idea to use the cache.

Please update `package.json` (replace `broccoli-writer` by `broccoli-caching-writer`).

**Comment by dfournier** [comment]
Test failed, please run it locally before your PR

**Comment by pieter-v** [comment]
David,

Sorry, forgot to adapt the tests

Regards
Pieter

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   ],
   "homepage": "https://github.com/dfournier/broccoli-requirejs",
   "dependencies": {
-    "broccoli-writer": "~0.1.1",
+    "broccoli-caching-writer": "^0.5.5",
```
Comment (dfournier) [inline package.json]: Please use `~` instead of `^`

**Comment by dfournier** [comment]
Looks good !

Before merging your PR I'd like you change `^` by `~` where I commented. Squash your commits to propose only one.

**Comment by dfournier** [comment]
Thanks for your work! It's available under version 1.1.0!

**Pull request merged**